cloudinary-core/src/test/java/com/cloudinary/AuthTokenTest.java

```diff
@@ -92,6 +93,16 @@ public void testAuthenticatedUrl() {
 
     }
 
+    @Test
+    public void testUrlAnalyticsWithQueryParams() {
+        cloudinary.config.analytics = true;
+        cloudinary.setAnalytics(new Analytics("F", "2.0.0", System.getProperty("java.version")));
+        cloudinary.config.privateCdn = true;
+        String url = cloudinary.url().signed(true).type("authenticated").generate("test");
+        assertEquals(url,"http://test123-res.cloudinary.com/image/authenticated/test?__cld_token__=st=11111111~exp=11111411~hmac=735a49389a72ac0b90d1a84ac5d43facd1a9047f153b39e914747ef6ed195e53");
```

Comment:
where is the analytics part in the URL?

Reply:
Since every url with query params will not have analytics I've removed it from this test as well

Reply:
maybe let's update the unit test name to something like `testUrlNoAnalyticsWithQueryParams `

Reply:
Done